test/e2e/send.feature

```diff
@@ -3,6 +3,9 @@ Feature: Send dialog
   Scenario: should allow to send when enough funds and correct address form
     Given I'm logged in as "genesis"
     And I fill in "1" to "amount" field
+    Then I should see "convertor" element
+    Given I wait 5 seconds
```

Comment:
I think the wait is unneccesary, because the next step calls `waitForElemAndMatchItsText` and the converter component puts the element into DOM only when the value loaded, so it will wait.